mayastor/src/core/bdev.rs

```diff
@@ -377,6 +377,13 @@ impl Bdev {
             Some(Bdev::from(bdev))
         }
     }
+    /// return share URI
+    pub fn get_share_uri(&self) -> Option<String> {
```

Comment:
How about extending the share trait? It already provides the ability to determine what it is shared with as well. This makes the code easier to reuse later on when we implement the trait for the nexus and replica's

Reply:
yeah that's a good idea :+1: